### PR TITLE
Complete the "React tutorial" project. 😎

### PR DESCRIPTION
**Pull request by TadesseDev**
# Note 📝
> - This project contains all (including optional) exercises from [react-tutorial-for-beginners](https://ibaslogic.com/react-tutorial-for-beginners/) exercise.
> - This repo has more than 14 branches that are made available to help people follow along on different sections of Exercises 
> - This pull request is based on the last committed branch 
# Changes introduced ✍🏼
---
- Install webpack with a CLI tool `create-react-app`. ✔
- Create react forms and raise and handle events. ✔
- Implement `CSS` in more than one way (inline CSS, the CSS stylesheet and the CSS Modules). ✔
- Add logic to persisting react state in local storage ✔
- Integrate React lifecycle methods. ✔
- Add `SVG` icons to the project navigation and actions. ✔
- Configure `React Router` to manage routing between pages in the app.✔
- Add a hamburger icon to hide and show the nav bar section. ✔
- Deploy live page.

> Feel free to make any suggestions... I'd love to improve my code so I can use your review. ✌

### CODE REVIEW

File: src/functionBased/components/ToDoContainer.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      return currentTask;
+    }));
+  };
+
+  return (
+    //  <Routes>
+    //     <Route path="/" element={} />
+    //     <Route path="/about" element={<About index />} />
+    //     <Route path="*" element={<NotMatch />} />
+    //   </Routes>
```
Comment (jsug9) [inline src/functionBased/components/ToDoContainer.js]: [OPTIONAL] Kindly remove the comments as they are no longer necessary.

**Review by jsug9 — APPROVED** [review]
Hi @Tadesse-Alemayehu, 

Excellent job on this project. Keep it up 🎆

Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Clean code ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] You can delete your class components, since you have branches, in case you need to check it again, you can always take a look at your branches.
- [ ] Your CSS code is great, however, advise you to try scss on you project, just to test how it works and be ready for future projects.

Cheers and Happy coding!👏👏👏

File: src/functionBased/components/ToDoContainer.js (hunk 1 above)
Comment (TadesseDev) [reply src/functionBased/components/ToDoContainer.js]: ### @jsug9 Nice catch 🙌. Will do.😎

**Comment by TadesseDev** [comment]
## Thank you @jsug9, I'll make sure to make the changes (Optional once) before I merge the branch. ✌

**Comment by TadesseDev** [comment]
## Dear @jsug9, I have completed the suggested (optional) changes in  https://github.com/Tadesse-Alemayehu/React-Tutorial-From-Scratch/pull/1/commits/a42adbdb0e88c8bb74f3240e4e05c15d9230842b commit hash. Feel free to suggest any changes you still feel could be improved.  ✌
>### Thank you 🙏

**Pull request merged**